src/index.js

```diff
@@ -18,11 +18,7 @@ function runPackageScripts({scriptConfig, scripts, args, options = {}}) {
     scripts = ['default']
   }
   const scriptNames = arrify(scripts)
-  if (options.parallel) {
-    return runParallel()
-  } else {
-    return runSeries()
-  }
+  return runSeries()
```

Comment:
if we do end up removing it we should probably open a refactoring issue to clean this type of things up :)